Tools/autotest/arduplane.py

```diff
@@ -1662,7 +1663,16 @@ def fly_soaring(self):
         self.arm_vehicle()
 
         # Enable thermalling RC
-        rc_chan = self.get_parameter('SOAR_ENABLE_CH')
+        rc_chan = 0
+        for i in range(8):
```

Comment:
I'll definitely be swiping this out into a `find_rc_channel_for_rc_option` at some stage :-)